src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+.product {
+  flex-wrap: wrap;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 40px;
+  max-width: 1200px;
+}
```

Comment:
you should create separate files for all bem block. you have product bem block styles inside `card.scss` now.
<img width="729" alt="image" src="https://user-images.githubusercontent.com/102411130/209292956-b7b2550d-24bb-4340-a104-91b5014f5559.png">